app.config.js

```diff
@@ -43,6 +44,7 @@ export default {
     eas: {
       projectId: config.EAS_PROJECT_ID,
     },
+    secretString: config.SECRET_STRING,
```

Comment:
this can't be hardcoded, the secret must be chosen by the user when exporting the secrets, so that only the user knows what is the password used to export them